h2/src/docsrc/help/help.csv

```diff
@@ -5281,7 +5285,11 @@ NEXTVAL ( [ schemaNameString, ] sequenceString )
 ","
 Returns the next value of the sequence.
 Used values are never re-used, even when the transaction is rolled back.
-If the schema name is not set, the current schema is used, and the sequence name is converted to uppercase (for compatibility).
+This method exists only for compatibility, it's recommended to use the standard
+[NEXT VALUE FOR sequenceName](https://h2database.com/html/grammar.html#sequence_value_expression)
```

Comment:
should this not be some kind of relative url?

Reply:
They are replaced with relative URLs during building of HTML/PDF documentation:
https://github.com/h2database/h2database/blob/413093a81a0fb21707d8740e9071dacd17618912/h2/src/tools/org/h2/build/doc/GenerateDoc.java#L294

There was a problem with relative URLs here, so I used absolute URLs with such replacement in #1949. I don't remember what was wrong with relative links.